### PR TITLE
New I18n langague support for "pt-BR"

### PR DESCRIPTION
**Pull request by kabrunko-dev**
## Checklist for challenge submission

- [ ] Start your PR message with Answer:${challenge_number}

### CODE REVIEW

File: .all-contributorsrc [hunk 1]
```diff
@@ -117,6 +121,13 @@
       "avatar_url": "https://avatars.githubusercontent.com/u/61125174?v=4",
       "profile": "https://github.com/ho-ssain",
       "contributions": ["doc"]
+    },
+    {
+      "login": "kabrunko-dev",
+      "name": "Diogo Nishikawa",
+      "avatar_url": "https://avatars.githubusercontent.com/u/142346548?v=4",
+      "profile": "https://github.com/kabrunko-dev/",
+      "contributions": ["translation-pt-BR"]
```
Comment (tomalaforge) [inline .all-contributorsrc:130]: You don't have to write this. 😅 It's done on a follow up PR and it's done automatically with all-contributors CLI.

**Comment by tomalaforge** [comment]
Very nice contribution, thanks a lot for your work.

**Review by tomalaforge — APPROVED** [review]

File: .all-contributorsrc (hunk 1 above)
Comment (kabrunko-dev) [reply .all-contributorsrc:130]: Hahaha sorry. I'm still new in the OSS world. But I'll keep it in mind now

Comment (tomalaforge) [reply .all-contributorsrc:130]: no problem 😂

**Pull request merged**